### ISSUE TITLE
TOC entries with no `href` just `title`

### ISSUE THREAD

**Issue by jccr** [readium/webpub-manifest#60]
When looking at data from the EPUB context: The EPUB spec [allows you to have navigation document headings be "link-less"](https://www.w3.org/publishing/epub3/epub-packages.html#confreq-nav-a).

For example:
```html
<ol>
  …
  <li>
    <span>Appendix</span>
    <ol>
      <li>
        <a href="appendix.xhtml#a.1-birds">A.1 Birds</a>
      </li>
      <li>
        <a href="appendix.xhtml#a.2-turtles">A.2 Turtles</a>
      </li>
    </ol>
  </li>
</ol>
```

How could someone represent this data as a RWPM ToC compact subcollection?
Given that the items are [`Link` objects that require an `href` attribute](https://readium.org/webpub-manifest/#24-the-link-object).

My only idea right now is to have the href value be a blank string.

The resulting collection would look like this:

```json
"toc": [
  {
    "href": "",
    "title": "Appendix",
    "children": [
      {
        "href": "appendix.xhtml#a.1-birds",
        "title": "A.1 Birds"
      },
      {
        "href": "appendix.xhtml#a.2-turtles",
        "title": "A.2 Turtles"
      }
    ]
  }
]
```

**Comment by jccr** [comment]
Another idea is that it takes the href without fragments.. somehow.. it's ambiguous...

The resulting collection would look like this:

```json
"toc": [
  {
    "href": "appendix.xhtml",
    "title": "Appendix",
    "children": [
      {
        "href": "appendix.xhtml#a.1-birds",
        "title": "A.1 Birds"
      },
      {
        "href": "appendix.xhtml#a.2-turtles",
        "title": "A.2 Turtles"
      }
    ]
  }
]
```

**Comment by qnga** [comment]
> My only idea right now is to have the href value be a blank string.

On mobile platforms, we arbitrarily use "#".

**Comment by mickael-menu** [comment]
> On mobile platforms, we arbitrarily use "#".

Yes, I felt somehow that at least it was a valid HREF, but it's not great. Maybe a blank string is better, I'm not sure.

> Another idea is that it takes the href without fragments.. somehow.. it's ambiguous...

I don't think that we can guarantee that all children are in the same resource though.

**Comment by HadrienGardeur** [comment]
Another option would be to drop this requirement for Link Objects listed in `toc`.
